tests/foreman/cli/test_gpgkey.py

```diff
@@ -108,37 +108,29 @@ def test_redmine_4272(self):
 
         self.assertEqual(gpg_key['content'], content)
 
-    @skip_if_bug_open('bugzilla', 1108227)
-    def test_bugzilla_1108227(self):
-        """@Test: Hammer fails to get a gpg info by name
+    def test_info_by_name(self):
+        """@Test: Create single gpg key and get its info by name
 
         @Feature: GPG Keys
 
-        @Assert: can get gpg key info by name
-
-        @BZ: 1108227
+        @Assert: specific information for GPG key matches the creation name
 
         """
         name = gen_string('utf8')
-        result = GPGKey.create({
-            'name': name,
-            'organization-id': self.org['id'],
-            'key': VALID_GPG_KEY_FILE_PATH,
-        })
-        self.assertEqual(result.return_code, 0)
-        self.assertEqual(len(result.stderr), 0)
 
         try:
-            new_obj = make_gpg_key(data)
+            new_obj = make_gpg_key({
+                u'key': VALID_GPG_KEY_FILE_PATH,
+                u'name': name,
+                u'organization-id': self.org['id'],
+            })
         except CLIFactoryError as err:
             self.fail(err)
 
-        # Can we find the new object?
         result = GPGKey().info({
-            'name': new_obj['name'],
-            'organization-id': self.org['id'],
+            u'name': new_obj['name'],
+            u'organization-id': self.org['id'],
         })
-
         self.assertEqual(result.return_code, 0)
         self.assertEqual(len(result.stderr), 0)
```

Comment:
@oshtaier actually what is in the log is stderr if any. We can aways check if something is in stderr even if the return code is 0. I think is better to leave that assertion there because it can prevent weird behaviour like returning 0 and having something in stderr.